src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java

```diff
@@ -133,6 +134,19 @@ public Boolean isKnownParticipant(
         return chronicleService.isKnownParticipant( studyId, participantId );
     }
 
+    @Override
+    public Void deleteParticipantAndAllNeighbors(
+            UUID studyId, String participantId, DeleteType deleteType ) {
+        // TODO implement this
+        return null;
+    }
+
+    @Override
+    public Void deleteStudyAndAllNeighbors( UUID studyId, DeleteType deleteType ) {
```

Comment:
Use `public void` and remove the `return null`.

Reply:
Same as https://github.com/openlattice/chronicle-server/pull/94#discussion_r432212819
chronicle api declares the method as `Void` instead of `void`

Reply:
`chronicle-api` has been updated. Please make the change.

https://github.com/openlattice/chronicle-api/pull/23

Reply:
Updated.